iphone/Classes/NetworkModule.m

```diff
@@ -286,11 +286,8 @@ - (void)application:(UIApplication *)application didRegisterForRemoteNotificatio
 {
   // called by TiApp
   if (pushNotificationSuccess != nil) {
-    NSString *token = [[[[deviceToken description] stringByReplacingOccurrencesOfString:@"<" withString:@""]
-        stringByReplacingOccurrencesOfString:@">"
-                                  withString:@""]
-        stringByReplacingOccurrencesOfString:@" "
-                                  withString:@""];
+    NSString *token = [NSString hexStringFromData:deviceToken];
```

Comment:
There is no such static method on NSString "hexStringFromData:" .

Reply:
Please use the following:
```objc
const unsigned *tokenBytes = [deviceToken bytes];
NSString *stringToken = [NSString stringWithFormat:@"%08x%08x%08x%08x%08x%08x%08x%08x",
                         ntohl(tokenBytes[0]), ntohl(tokenBytes[1]), ntohl(tokenBytes[2]),
                         ntohl(tokenBytes[3]), ntohl(tokenBytes[4]), ntohl(tokenBytes[5]),
                         ntohl(tokenBytes[6]), ntohl(tokenBytes[7])];
 ```